spec/requests/embedded_shopfronts_headers_spec.rb

```diff
@@ -43,16 +43,19 @@
 
     context "with a valid whitelist" do
       before do
-        Spree::Config[:embedded_shopfronts_whitelist] = "test.com"
+        Spree::Config[:embedded_shopfronts_whitelist] = "example.com external-site.com"
+        allow_any_instance_of(ActionDispatch::Request).to receive(:referer).and_return('http://www.external-site.com/shop?embedded_shopfront=true')
```

Comment:
You can have a reference to the actual request object from `controller.request`. That's more concise and less verbose IMO.

Reply:
I had real trouble getting these extra tests to work. In this instance, the `controller.request` object is not available, as we are viewing the page "through" an iframe. We can see what's in the iframe, but not access the controller directly. I know it's a weird spec, but I couldn't see any other way to test how an iframe loads content from another page.

Reply:
:+1: